state-chain/runtime/src/lib.rs

```diff
@@ -2072,8 +2072,41 @@ impl_runtime_apis! {
 		fn cf_validate_refund_params(retry_duration: u32) -> Result<(), DispatchErrorWithMessage> {
 			pallet_cf_swapping::Pallet::<Runtime>::validate_refund_params(retry_duration).map_err(Into::into)
 		}
+
+		fn cf_open_btc_deposit_channels(account_id: AccountId) -> Result<Vec<<cf_chains::Bitcoin as cf_chains::Chain>::ChainAccount>, DispatchErrorWithMessage> {
```

Comment:
Can we make the account_id optional - we might want to get all channels irrespective of who opened them.